server/routes/index.js

```diff
@@ -1,8 +1,12 @@
-var express = require("express");
-var router = express.Router();
+let express = require("express");
+let router = express.Router();
+let addUser = require("../db")
 
-router.get("/welcome", function(req, res, next) {
-  res.status(200).send({ welcomeMessage: "Step 1 (completed)" });
+router.post("/", (req, res, next) => {
```

Comment:
this should be in a different route, and you need to validate this data before adding a user